docs/docs/dev/explore/observability.md

```diff
@@ -223,9 +223,11 @@ This dashboard lets you investigate the status of a single dataplane in the mesh
 ##### Kuma Mesh
 
 This dashboard lets you investigate the aggregated statistics of a single mesh.
+It provides a topology view of your service traffic dependencies (**Service Map**)
+and includes information such as number of requests and error rates.
 
 <center>
-<img src="/images/docs/1.1.2/grafana-dashboard-kuma-mesh.jpg" alt="Kuma Mesh dashboard" style="width: 600px; padding-top: 20px; padding-bottom: 10px;"/>
+<img src="/images/docs/1.8.0/grafana-dashboard-kuma-mesh.png" alt="Kuma Mesh dashboard" style="width: 600px; padding-top: 20px; padding-bottom: 10px;"/>
```

Comment:
please convert the image to JPG. It was 73KB before, now it's 508KB.

I think recently MacOS switched the default screenshot format to PNG. You can set this to JPG again
https://www.laptopmag.com/articles/change-default-screenshot-file-format-macos

Reply:
I made the screen smaller (similar in size as it was before), as changing it to json didn't make it much smaller in size (maybe few percents)